src/message.rs

```diff
@@ -18,6 +18,22 @@ pub struct Message {
     pub filename: String,
 }
 
+impl Message {
+    pub fn from_check(filename: String, check: Check) -> Self {
+        let mut location = check.location;
+        location.go_right();
+        let mut end_location = check.end_location;
+        end_location.go_right();
```

Comment:
this doesn't feel right 🙈

Reply:
I think you can use Location::new and location.row() methods to create a new location here! (On subway so low-fidelity message…)

Reply:
Ohh that makes more sense hah I was too influenced by Go in my first attempt. Pushed a fix!

Reply:
I think you can use Location::new and location.row() methods to create a new location here! (On subway so low-fidelity message…)

Reply:
Woo, thanks!